### PR TITLE
Adding a test to extract_partitions: whitespace in fasta sequence header...

### PR DESCRIPTION
**Pull request by lexnederbragt**
Addresses #175

- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Is the Copyright year up to date?

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, ok to test

File: tests/test_scripts.py [hunk 1]
```diff
@@ -1298,6 +1298,35 @@ def test_extract_partitions():
...
+    in_dir = os.path.dirname(graphbase)
+
+    # get the final part file
+    partfile = os.path.join(in_dir, 'test-overlap2.fa.part')
+
+    # ok, now run extract-partitions.
+    script = scriptpath('extract-partitions.py')
+    args = ['extracted', partfile]
+
+    runscript(script, args, in_dir)
```
Comment (mr-c) [inline tests/test_scripts.py]: This needs to be put back to the `utils.runscript(...` version.

**Comment by lexnederbragt** [comment]
Working on the checklist:

* how do I do a spellcheck?
* Copyright year: if I do `git grep Copyright` I get lot's of 2009-2013 and 2009-2014, so i hope that's not what you mean...

**Comment by mr-c** [comment]
@lexnederbragt I enjoy https://github.com/lucasdemarchi/codespell and our dev docs also recommend aspell:

"GNU aspell can also be used to check the spelling in a single file:

`aspell check --mode ccpp $filename`"
http://khmer.readthedocs.org/en/latest/dev/coding-guidelines-and-review.html#coding-standards

WIth regards to the copyright year: if you've updated a file the copyright years should include the current year.

**Comment by mr-c** [comment]
Looks like you are good to go! Merging now.

**Comment by mr-c** [comment]
@lexnederbragt Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.
